Base/etc/SystemServer.ini

```diff
@@ -51,6 +51,10 @@ SystemModes=text,graphical
 MultiInstance=true
 AcceptSocketConnections=true
 
+[NetworkServer]
+User=root
+SystemModes=text,graphical,self-test,generate-manpages
```

Comment:
Why would `generate-manpages` need networking? :thinking:

Reply:
I just copied that from LookupServer previously, fixed! :)